docs/updating-NuGet-zip-file.md

```diff
@@ -0,0 +1,19 @@
+# Updating NuGet ZIP file
+
+The CI restore works by downloading the contents of the packages directory from Azure directly.  Hence if a package is updated  this zip will need to be rebuilt.  
+
+This is done by executing the following on a Windows box.  
+
+<!-- XXX: Does the following mean something?  If so, explain it better:
+- Change to the root of the enlistment.
+-->
+
+- Delete or rename `~\.nuget`
+- Run `Restore.cmd`
```

Comment:
Change this to "from the root of the repository, run Restore.cmd" to replace the "change to the root of the enlistment" comment above.